app/src/main/java/com/example/concordia_campus_guide/Global/User.java

```diff
@@ -4,10 +4,20 @@
 
 import java.util.List;
 
+
+//@Entity (tableName = "users")
 public class User {
+
+  //  @Ignore
     public static User instance;
+
+    //@ColumnInfo (name ="id")
     private long id;
+
+
     private List schedule;
+
+    //@Embedded
```

Comment:
can these comments be removed?

Reply:
since the User is still a work in progress, this is a beginning of work that is on hold and I don't want to delete it since Hampic knows more about this